src/Nethermind/Nethermind.HealthChecks/IClHealthTracker.cs

```diff
@@ -0,0 +1,11 @@
+// SPDX-FileCopyrightText: 2025 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+
+namespace Nethermind.HealthChecks;
+
+public interface IClHealthTracker : IAsyncDisposable
```

Comment:
Not sure about having `IAsyncDisposable` in the interface itself since "being disposable" is up to each implementation.